doc/doxygen/tutorial/CMakeLists.txt

```diff
@@ -47,14 +65,17 @@ ADD_CUSTOM_COMMAND(
   ARGS
     ${CMAKE_SOURCE_DIR}/doc/doxygen/scripts/steps.pl
     ${CMAKE_CURRENT_SOURCE_DIR}/tutorial.h.in
-    ${DEAL_II_STEPS}
+    ${_deal_ii_steps}
+    ${_code_gallery_names}
     > ${CMAKE_CURRENT_BINARY_DIR}/tutorial.h
   DEPENDS
-    ${DEAL_II_STEPS}
-    ${DEAL_II_STEPS_BUILDSON}
-    ${DEAL_II_STEPS_KIND}
-    ${DEAL_II_STEPS_TOOLTIP}
+    ${CMAKE_SOURCE_DIR}/doc/doxygen/scripts/steps.pl
     ${CMAKE_CURRENT_SOURCE_DIR}/tutorial.h.in
+    ${_deal_ii_steps}
+    ${_deal_ii_steps_kind}
+    ${_deal_ii_steps_tooltip}
+    ${_deal_ii_steps_buildson}
+    ${_code_gallery_names}
```

Comment:
If I recall correctly: The dependencies ```${_deal_ii_steps}``` and ```${_code_gallery_names}``` do exactly nothing because these lists contain directories.